tests/framework/extensions/psact/createdeployment.go

```diff
@@ -57,7 +57,7 @@ func CreateNginxDeployment(client *rancher.Client, clusterID string, psact strin
 
 		deploymentResp, err := steveclient.SteveType(workloads.DeploymentSteveType).ByID(deploymentTemplate.Namespace + "/" + deploymentTemplate.Name)
 		if err != nil {
-			return false, err
+			return false, nil
```

Comment:
nit; is it worth adding a comment here on why we aren't returning the error?

Reply:
For sure I can do that. I need make some updates, as it still flakey